ddp-workspace/projects/ddp-rarex/src/assets/i18n/en.json

```diff
@@ -314,7 +314,8 @@
   "ParticipantsList": {
     "Title": "Participant Dashboard",
     "NewParticipant": "New Participant",
-    "NoParticipants": "You don't have any participants at this moment. You can add one by clicking the button below."
+    "NoParticipants": "You don't have any participants at this moment. You can add one by clicking the button below.",
+    "DeleteError": "This participant has already consented and can’t be deleted here. If you wish to withdraw consent, please contact Rare-X study personnel directly"
   },
```

Comment:
The message as agreed in Slack

Reply:
Let's put a `.` (period) at the end there the close off that sentence?